src/Frontend/Core/Engine/Model.php

```diff
@@ -326,6 +329,29 @@ public static function getModuleSettings($module)
     }
 
     /**
+     * Creates router for module based on routing file
+     *
+     * @param $module
+     * @return null|Router
+     */
+    public static function getModuleRouter($module)
```

Comment:
Or we could load all modules into s2 container. So we could reach them with $this->getContainer()->get('Blog') or something like that.